### ISSUE TITLE
ncnn Cast  not support int32_to_int64 . onnx reshape need it.

### ISSUE THREAD

**Issue by laobadao** [Tencent/ncnn#1108]
Because tensorflow is not supported, I convert TF Pb to onnx by tf2onnx (tensorflow2onnx), include BasicLSTMCell, and one BasicLSTMCell op split to multiple ops in onnx, like Cast Split Sigmoid Tanh and so on. 
Cast need int32_to_int64 for reshape, Can you fix it?

**Comment by nihui** [comment]
The onnx file converted from tensorflow model is probably not supported either
because I have zero knowledge of the tensorflow usecases
You would better training your model with caffe/mxnet/pytorch

**Comment by nihui** [comment]
针对onnx模型转换的各种问题，推荐使用最新的pnnx工具转换到ncnn
In view of various problems in onnx model conversion, it is recommended to use the latest pnnx tool to convert your model to ncnn

```shell
pip install pnnx
pnnx model.onnx inputshape=[1,3,224,224]
```

详细参考文档
Detailed reference documentation
https://github.com/pnnx/pnnx
https://github.com/Tencent/ncnn/wiki/use-ncnn-with-pytorch-or-onnx#how-to-use-pnnx
